src/js/components/TextInput/StyledTextInput.js

```diff
@@ -9,9 +10,23 @@ import {
 } from '../../utils';
 import { defaultProps } from '../../default-props';
 
+const getPlainStyle = plain => {
+  if (plain === 'full') {
+    return css`
+      ${plainInputStyle} padding: 0;
+    `;
+  }
+  return plain && plainInputStyle;
+};
+
+const plainPropType = PropTypes.oneOfType([
+  PropTypes.bool,
+  PropTypes.oneOf(['full']),
+]);
+
```

Comment:
Why are propTypes defined on the Styled file? this is not consistent with other components, it is usually kept on the doc.js file.

Reply:
Moved propTypes to doc.js in latest push